src/fr/contenu-editorial/generalites.md

```diff
@@ -5,48 +5,47 @@ title: "Recommandations éditoriales générales"
-
-À vérifier&nbsp;: 
-
-- Globalement, ne pas utiliser de mise en page complexe de tableau (fusion de cellules d’un tableau, multicolonnage…).
-- Éviter d’imbriquer les tableaux de mise en page (grand maximum, 2 niveaux d’imbrication), mais aussi&nbsp;:
-- Ne pas fusionner les cellules d’un tableau, mais utiliser plusieurs tableaux plus simples.
-- Ne pas conserver de cellules vides.
-- Ne pas ajouter d’images de fond.
-- Assurer une lecture séquentielle (on doit pouvoir lire de gauche à droite et de haut en bas).
-- Éviter les tableaux de données, mais aussi les graphiques ou les schémas complexes.
+* Aligner les textes à gauche (éviter de justifier). 
+* Mettre une majuscule à chaque début de phrase mais éviter les phrases entières en majuscule.
+* Éviter l’italique.	
+*	Utiliser des polices simples, sans serif : Arial, Calibri ou la <a href="https://opensource.orange.com/fr/category/actualites/ow2-fr/osai-fr/accessible_dfa-fr/">police Accessible DfA</a> ayant une taille au moins égale à 10 points.
+*	Ajouter des accents aux majuscules éèêçàïùÉÈÊÇÀÏÙ («&nbsp;Insertion&nbsp;&gt; Symbole&nbsp;&gt; Texte normal&nbsp;» ou utiliser [les raccourcis clavier](#annexes)).
+*	Mettre en place une ponctuation simple.
+*	Éviter les fonds texturés (images).
+* Ne pas faire de retours chariots répétés ou de tabulation pour espacer (mais la fonction «&nbsp;Paragraphe&nbsp;&gt; Espacement&nbsp;» ou «&nbsp;Retrait&nbsp;» dans Word, par exemple).
+*	Éviter les tableaux de mise en page ou tableaux de données complexes (sauf en HTML si la [structure du tableau est accessible](/fr/web/developper/contenu-textuel/#structurer-les-tableaux-de-donnees)). 
+*	Éviter le contenu "image" porteur d’information : graphes, schémas, diagrammes (sauf en HTML si une [alternative textuelle est accessible](/fr/web/designer/contenu-non-textuel/))
+*	Utiliser des tournures de phrases limpides, des phrases courtes et à la construction simple&nbsp;: sujet, verbe, complément. Une idée par phrase.
+*	Éviter au maximum les mots trop compliqués ou spécifiques à un domaine, et sinon, les expliquer.
+*	Utiliser le présent, de préférence le style direct et la forme active (pas de subjonctif, pas de conjonction).
+*	Utiliser les caractères typographiques du Français&nbsp;: tiret - , cadratin — , «&nbsp;guillemets français&nbsp;».
+*	Éviter les termes dans une langue étrangère (mots en anglais), les sigles ou abréviations sauf ceux passés dans le langage courant.
+*	Mettre en place un indicateur de fin de document (par exemple, une image indiquant «&nbsp;fin de mail&nbsp;», un texte blanc sur fond blanc «&nbsp;fin de document&nbsp;»).
+
+## Mise en page avec des tableaux 
+
+Sachant que les clients de messagerie (Outlook inclus) obligent encore parfois à utiliser une [mise en page en tableaux](/fr/contenu-editorial/emails/) : ajouter un role=presentation à vos balises table afin que les synthèses vocales comprennent qu'il ne s'agit que de mise en forme.  
```

Comment:
s/role=presentation/attribut `role="presentation`/

Reply:
Fixed in c216677282df01aa3e077a45ded6d6e8e62eff4d